rosa_getting_started/rosa-getting-started.adoc

```diff
@@ -1,11 +1,16 @@
 :_content-type: ASSEMBLY
 [id="rosa-getting-started"]
-= Getting started with {product-title}
+= Comprehensive guide to getting started with {product-title}
 include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-getting-started
 
 toc::[]
 
+[NOTE]
+====
+If you are looking for a quickstart guide for ROSA, see xref:../rosa_getting_started/rosa-quickstart.adoc#rosa-quickstart[{product-title} quickstart guide].
+====
+
 Follow this getting started document to quickly create a {product-title} (ROSA) cluster, grant user access, deploy your first application, and learn how to revoke user access and delete your cluster.
```

Comment:
If this is the non-quickstart guide to creating a cluster, I'd recommend taking "quickly" out of the sentence "Follow this getting started document to quickly create..." just so there's no confusion over which one is quicker.